.github/workflows/ci.yml

```diff
@@ -2,35 +2,42 @@ name: CI
 on: [push, pull_request]
 jobs:
   tests:
-    name: Tests
-    runs-on: ${{ matrix.os }}
+    runs-on: 'ubuntu-latest'
+    name: PHP ${{ matrix.php }} - Laravel ${{ matrix.laravel }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
```

Comment:
why drop testing on macOS?

my dev is done on macOS though production is debian

Reply:
None of [doctrine/migrations](https://github.com/doctrine/migrations/blob/3.0.x/.github/workflows/continuous-integration.yml), [doctrine/orm](https://github.com/doctrine/orm/blob/2.8.x/.github/workflows/continuous-integration.yml), and [laravel/framework ](https://github.com/laravel/framework/blob/8.x/.github/workflows/tests.yml)tests with macos. Seems weird that we should support it.

Reply:
Fair enough